### ISSUE TITLE
Titlebar uses theme colors in single-tab mode

### ISSUE THREAD

**Issue by Mukul1127** [Night-Star04/vscode-fluent-ui#17]
The theme you are using can bleed in and the controls are getting clipped.
![{BB990A1A-B33E-4E13-9168-D3385E610A81}](https://github.com/user-attachments/assets/db69ffc2-4f43-4e10-b143-c8cd73888aad)
Solution CSS:
```
#workbench\.parts\.editor > div.content > div > div > div > div > div.monaco-scrollable-element > div.split-view-container > div > div > div.title.show-file-icons > div.title-actions {
  margin-right: 30px;
}

#workbench\.parts\.editor > div.content > div > div > div > div > div.monaco-scrollable-element > div.split-view-container > div > div > div.title.show-file-icons {
	background: var(--noise-bg) repeat var(--editor-bg) !important;
}
```

**Comment by Night-Star04** [comment]
Dear @Mukul1127,

Thank you for submitting this issue and providing the CSS solution! I apologize for the delayed response.

Regarding the Titlebar looks weird in single tab mode issue, I would need some additional information to better understand the situation and proceed with a fix:

1. Theme in use:

    Could you please share the name of the VSCode theme you were using? This will help me replicate the issue.

2. System and VSCode version:

    Your operating system (Windows/macOS/Linux) and its version.

    The version of VSCode you’re currently using.

3. Steps to reproduce:

    Besides the single tab mode, are there other conditions (e.g., window size, specific settings) that trigger this issue?

4. Screenshots or videos:

    If you have more screenshots or a video showing the problem, it would greatly help me understand and pinpoint the issue.

Thank you again for your patience! Once I receive more information, I’ll prioritize resolving this and provide an update as soon as possible.

Best regards,
**Night Star**

**Comment by Mukul1127** [comment]
Hi,

I was using "Github Dark Default" from the official Github theme extension on windows. VSCode details below,
```
Version: 1.96.2
Release: 24355
Commit: 29b115fb6fb418b80e93102cb177e06803e7b907
Date: 2024-12-21T00:46:06.606Z
Browser: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) VSCodium/1.96.2 Chrome/128.0.6613.186 Electron/32.2.7 Safari/537.36
```
For steps to reproduce, not really any that I know of, it just uses theme's colors instead of the mica background when you swap to one tab mode. Here's a video: (Too big to upload on Github)
https://imgur.com/a/aoohDJp

**Comment by Night-Star04** [comment]
Hi, Thank you for providing the video. 
Since I don’t usually use the single tab mode, I wasn’t able to notice this issue before. However, I now understand the problem and will work on fixing it as soon as possible.

**Comment by Night-Star04** [comment]
After studying the div of this tab, is it possible to use the following css?
```css
.monaco-workbench .part.editor > .content .editor-group-container > .title.breadcrumbs {
    background: var(--card-bg) !important;
    padding-right: var(--s5) !important;
}
```

**Comment by Mukul1127** [comment]
I tried using that css using custom css and js loader but it didn't seem to change anything, maybe check the selector?

**Comment by Night-Star04** [comment]
emm... It should not be unresponsive. I using that css using custom css and js extension too. See the picture below.

![螢幕擷取畫面 2025-01-03 115443](https://github.com/user-attachments/assets/616cef68-cc81-48a5-a0a0-86de7a4840ec)

Vscode details:
```
Version: 1.96.2 (user setup)
Commit: fabdb6a30b49f79a7aba0f2ad9df9b399473380f
Date: 2024-12-19T10:22:47.216Z
Electron: 32.2.6
ElectronBuildId: 10629634
Chromium: 128.0.6613.186
Node.js: 20.18.1
V8: 12.8.374.38-electron.0
OS: Windows_NT x64 10.0.22631
```

**Comment by Night-Star04** [comment]
I just found out during verification that the background color of the path is different. Can you help me test it too?

```
.monaco-workbench
    .part.editor
    > .content
    .editor-group-container
    > .title:not(.tabs)
    > .label-container
    .monaco-breadcrumbs {
    background-color: transparent !important;
}
```

![image](https://github.com/user-attachments/assets/f4a4b146-8f06-43fe-b600-8f9024de0892)

**Comment by Mukul1127** [comment]
![image](https://github.com/user-attachments/assets/322d6447-5ce5-4d02-9e4c-42f35e3e935d)
I don't know why but for me, the path's background doesn't inherit the theme's background. Could this be because I am using VSCodium?

**Comment by Night-Star04** [comment]
I see, VS Code and VSCodium might have slight differences in certain cases. However, the `.title:not(.tabs)` method works properly on both.

As for the Path's background, the current version sets `.monaco-breadcrumbs` to use `var(--card-bg)`. You can refer to [this link](https://github.com/Night-Star04/vscode-fluent-ui/blob/v4.4.0/src/css/editor_chrome.css#L497) for more details.

**Comment by Mukul1127** [comment]
So are we done here or is there anything else? We both agree on the `.title:not(.tabs)` method.

**Comment by Night-Star04** [comment]
Yes, this part is mostly completed. I will finish the remaining parts as soon as possible. 
If there are any other adjustments needed for the single-tab mode, feel free to let me know.
